code/provision/rke/cluster.yml

```diff
@@ -0,0 +1,79 @@
+# If you intened to deploy Kubernetes in an air-gapped environment,
```

Comment:
Would we commit this file in the end as I guess it's auto generated? I think it's worth doing anyway even if we don't edit it manually as we can use it ourselves.

Reply:
So I have manually created the cluster.yml when I did this. There is a rke-kubernetes role that generates it dynamically now. So we won't have to commit the file